script/check_availability.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+# This script try to connect to elastic search and access the DB
+# return 0 if success / return 1 if error
+
+ASSEMBL_ROOT=$VIRTUAL_ENV/..
+LOCAL_INI=$ASSEMBL_ROOT/local.ini
+
+# parse elasticsearch_host from local.ini
+elasticsearch_host="$(grep "elasticsearch_host" $LOCAL_INI | tr -d ' ' | awk -F "=" '{print $2}')"
+
+# parse elasticsearch_port from local.ini
+elasticsearch_port="$(grep "elasticsearch_port" $LOCAL_INI | tr -d ' ' | awk -F "=" '{print $2}')"
+
+# parse elasticsearch_index from local.ini
+elasticsearch_index="$(grep "elasticsearch_index" $LOCAL_INI | tr -d ' ' | awk -F "=" '{print $2}')"
+
+# parse elasticsearch_version from local.ini
+elasticsearch_version="$(grep "elasticsearch_version" $LOCAL_INI | tr -d ' ' | awk -F "=" '{print $2}')"
+
+ELASTICSEARCH_URL=http://${elasticsearch_host}:${elasticsearch_port}
+
+CURL="$(curl --fail --silent --show-error ${ELASTICSEARCH_URL} 2>&1)"
+
+# if curl didn't succeed 
+if [ $? != 0 ]; then
+    echo "Failed to connect to Elaticsearch server" >> $ASSEMBL_ROOT/var/log/assembl.log
```

Comment:
Actually, since the changes made in a1d811fdf450b591417c1990dce00f4759f43cc3, can simply do a printf statement and we'll have logs of it in the stdout. So, 
```suggestion
printf "Failed to connect to Elaticsearch server with failure: \t %s\n" "$CURL")
```